packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -80,6 +80,7 @@ export default class Server {
 
     constructor() {
         this.connection.onInitialize(this.onInitialize.bind(this));
+        this.connection.onInitialized(this.onInitialized.bind(this));
```

Comment:
Looks like config values are not available during `onInitialize` which is why I'm using `onInitialized` instead.

Once the feature is GA, we can move everything back to `onInitialize`.